### DISCUSSION TITLE
Is there any guide to use and bundle client component?

### DISCUSSION THREAD

**Discussion by fajaralfa** [honojs/hono#3111] [Q&A]
Hi, currently i bundle client component using esbuild directly and the output is ~24kb minified.
is there any way to use it so that i can get like the docs said:
> The counter program in hono/jsx/dom is only 2.8KB with Brotli compression. But, 47.8KB for React.

Thanks.

**Comment by EdamAme-x** [comment]
about Brotli compression?
or about how to use **hono/jsx**?

if you want to use **hono/jsx**.

Sample is here: https://hono.dev/docs/guides/jsx-dom and you should add `jsxImportSource: "hono/jsx"` at **tsconfig.json**

**Reply by EdamAme-x** [reply]
https://github.com/yusukebe/hono-jsx-dom-examples
